modules/kubernetes-addons/kyverno/main.tf

```diff
@@ -0,0 +1,35 @@
+resource "kubernetes_namespace_v1" "this" {
+  metadata {
+    name = local.kyverno_helm_config["namespace"]
+    labels = {
+      "app.kubernetes.io/managed-by" = "terraform-aws-eks-blueprints"
+    }
+  }
+}
+module "kyverno_helm_addon" {
+  count             = var.enable_kyverno ? 1 : 0
+  source            = "../helm-addon"
+  manage_via_gitops = var.manage_via_gitops
+  helm_config       = local.kyverno_helm_config
+  irsa_config       = null
+  addon_context     = var.addon_context
+  depends_on        = [kubernetes_namespace_v1.this]
+}
+module "kyverno_policies_helm_addon" {
+  count             = var.enable_kyverno_policies ? 1 : 0
+  source            = "../helm-addon"
+  manage_via_gitops = var.manage_via_gitops
+  helm_config       = local.kyverno_policies_helm_config
+  irsa_config       = null
+  addon_context     = var.addon_context
+  depends_on        = [kubernetes_namespace_v1.this, module.kyverno_helm_addon]
+}
+module "kyverno_ui_helm_addon" {
+  count             = var.enable_kyverno_ui ? 1 : 0
+  source            = "../helm-addon"
+  manage_via_gitops = var.manage_via_gitops
+  helm_config       = local.kyverno_ui_helm_config
+  irsa_config       = null
+  addon_context     = var.addon_context
+  depends_on        = [kubernetes_namespace_v1.this, module.kyverno_helm_addon, module.kyverno_policies_helm_addon]
```

Comment:
does the reporter rely on policies? 

what happens to the deployment if they are not deployed in order? we want to avoid explicit dependencies as much as possible

Reply:
Reporter and Policies can work independently. I doesn't need to be in order.